terraform-organize-configuration-workshop/courseBase.sh

```diff
@@ -0,0 +1,7 @@
+mkdir -p learn-terraform/assets
+
+cp index.html learn-terraform/assets/
```

Comment:
```suggestion
cp /root/learn-terraform/index.html /root/learn-terraform/assets/
```
Use full paths for reliability of you are using the assets feature.